src/Service/NotificationService.php

```diff
@@ -80,4 +81,42 @@ public function notifyUsers(
             );
         }
     }
+
+    /**
+     * @param \App\Model\Entity\User $fromUser User who did gib the potato
+     * @param string $taggedMessageId id of the tagged message
+     * @param \App\Event\MessageEvent $event The event.
+     * @return void
+     */
+    public function notifyChannelNewQuickwin(
+        User $fromUser,
+        string $taggedMessageId,
+        MessageEvent $event,
+    ): void {
+        $blocks = [
+            [
+                'type' => 'section',
+                'text' => [
+                    'type' => 'mrkdwn',
+                    'text' => "<@{$fromUser->slack_user_id}> posted a new <" . $event->permalink . '|#quickwin>!',
+                ],
+            ],
+            [
+                'type' => 'divider',
+            ],
+            [
+                'type' => 'section',
+                'text' => [
+                    'type' => 'mrkdwn',
+                    'text' => '<' . Router::url('/quickwins', true) . '?id=' . $taggedMessageId . '|Visit Hall of Fame>',
```

Comment:
Is this done because accessing a query parameter in the FE is easier?
If not, I would prefer a `/quickwins/<uui>` URL.